app/src/organisms/ProtocolSetup/LabwarePositionCheck/hooks/__tests__/useIntroInfo.test.tsx

```diff
@@ -6,7 +6,7 @@ import { createStore, Store } from 'redux'
 import { renderHook } from '@testing-library/react-hooks'
 import { getPipetteNameSpecs, PipetteName } from '@opentrons/shared-data'
 import { getPipetteMount } from '../../../utils/getPipetteMount'
-import { useProtocolDetails } from '../../../../RunDetails/hooks'
+import { useProtocolDetailsForRun } from '../../../../Devices/hooks/useProtocolDetailsForRun'
 import * as hooks from '..'
```

Comment:
i had the same test failure in my LPC update branch, this should fix:
```suggestion
import { useIntroInfo } from '../useIntroInfo'
```